indi-eqmod/eqmodbase.cpp

```diff
@@ -1069,7 +1069,7 @@ bool EQMod::ReadScopeStatus()
                 currentDEEncoder = mount->GetDEEncoder();
                 parkRAEncoder    = GetAxis1Park();
                 parkDEEncoder    = GetAxis2Park();
-                if( static_cast<int32_t>(parkRAEncoder - currentRAEncoder) != 0 || static_cast<int32_t>(parkDEEncoder - currentDEEncoder) != 0 ) {
+                if( parkRAEncoder != currentRAEncoder || parkDEEncoder != currentDEEncoder ) {
```

Comment:
How about we use std::abs(a - b) > threshold ? If the different is like 1 tick, should we really issue a slew command? there should be an acceptable minimal threshold. maybe 5 or 10 ticks

Reply:
Done, but why are the builds failing in a class which was not change in my branch?